gmail/src/models/error_message.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Represent an error and translate its code to a message.
+ */
+
+const _ERROR_CODE_MESSAGES: Record<string, string> = {
+    http_error_odoo: "Could not connect to database. Try to log out and in.",
```

Comment:
These should probably be uppercased as well.

Reply:
The things is that Odoo or IAP return those value in lower case, so it's more like an "error code" than a constant